components/activeLink.js

```diff
@@ -0,0 +1,51 @@
+import { useRouter } from "next/router";
+
+const ActiveLink = () => {
+  const handleClick = (e) => {
+    const router = useRouter();
+    e.preventDefault();
+    router.push(href);
+  };
+  return (
+    <>
+      <span>
+        <Link href="/">
+          <button
```

Comment:
Avoid using a button element. These are designed for forms and toolbar interfaces. Instead use an a tag.